certified-connectors/Telnyx SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,260 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Telnyx",
+    "description": "Deliver SMS messages over the Telnyx network to ensure your notifications reach the intended recipient in a fast, reliable, and cost effective manner.",
+    "version": "1.0",
+    "contact":{
+	"name": "Telnyx Support",
+	"url": "https://support.telnyx.com/",
+	"email": "support@telnyx.com"
+    }
+  },
+ "x-ms-connector-metadata": [
+      {
+        "propertyName": "Website",
+        "propertyValue": "https://www.telnyx.com"
+      },
+      {
+        "propertyName": "Privacy policy",
+        "propertyValue": "https://telnyx.com/privacy-policy"
+      },
+      {
+        "propertyName": "Categories",
+        "propertyValue": "Communication;Internet of Things"
+      }
+  ], 
+  "host": "api.telnyx.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v2/messages": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",

```

Comment:
@fabianp5060 can you please provide a more detailed description for 2xx response codes? You can refer to [this example](https://docs.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).